static/css/exception_error.css

```diff
@@ -3,14 +3,16 @@ html {
 }
 
 body {
-    background-image: url("/static/img/Exception_BackGround.jpeg");
+    background-image: url("/static/img/BackGround.jpg");
```

Comment:
@pisces860225  Is the SCSS designing on the roadmap?

Reply:
OK !
This idea will be included in future feature improvements.